imap_processing/cli.py

```diff
@@ -432,16 +431,16 @@ def post_processing(
 
         logger.info("Writing products to local storage")
 
-        list_of_files = [
-            dep_obj.filename_list for dep_obj in dependencies.processing_input
-        ]
-        list_of_files = np.array(list_of_files).flatten()
-        logger.info("Parent files: %s", list_of_files)
+        # Parent files used to create these datasets
+        # https://spdf.gsfc.nasa.gov/istp_guide/gattributes.html.
+        parent_files = [p.name for p in dependencies.get_file_paths()]
```

Comment:
@tech3371 is this the same as before but simplified with your recent refactor to use `get_file_paths()`? I think this is doing what we want and will include all ancillary files etc... Is that what we want?

Reply:
:+1: That sounds reasonable to me to add a helper function that would extract those files as well and inject here later once that is set up.